admin/roles/class-role-manager-vip.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * @package WPSEO\Admin\Roles
+ */
+
+/**
+ * VIP implementation of the Role Manager.
+ */
+final class WPSEO_Role_Manager_VIP extends WPSEO_Abstract_Role_Manager {
+	/**
+	 * Adds a role to the system.
+	 *
+	 * @param string $role         Role to add.
+	 * @param string $display_name Name to display for the role.
+	 * @param array  $capabilities Capabilities to add to the role.
+	 *
+	 * @return void
+	 */
+	protected function add_role( $role, $display_name, array $capabilities = array() ) {
+		$enabled_capabilities  = array();
+		$disabled_capabilities = array();
+
+		// Build lists of enabled and disabled capabilities.
+		foreach ( $capabilities as $capability => $grant ) {
+			if ( $grant ) {
+				$enabled_capabilities[] = $capability;
```

Comment:
Adding a `continue` after adding the capability would allow you to remove the next `if`-statement.

Reply:
Preferring explicitly writing what is happening and why. Find it much more readable with two if-statements instead of a continue.